opus/application/static_media/js/cart.js

```diff
@@ -8,7 +8,31 @@
 
 // The download options left pane will become a slide panel when screen width
 // is equal to or less than the threshold point.
-const cartLeftPaneThreshold = 940;
+const cartLeftPaneThreshold = 1160;
+// Max height for the contents of download links history (.popover-body) before we enable PS.
+const downloadLinksPBMaxHeight = 200;
+// Html string for customized popover window. The reason we don't put the whole html
+// element in DOM first and retrieve by .html() later is because we need to attach
+// ps to .popover-body, and by adding the whole popover element in html first, there
+// will be two .popover-body elements (one from the inserted DOM at the beginning and
+// one from actual popover window). Jquery selector will only select the first
+// .popover-body and failed to attached the ps to the actual popover window. (Note: there
+// is no proper way to distinguish between them because the 2nd one is the duplicate of
+// the 1st one).
+const downloadLinksPopoverTemplate = "<div class='popover' role='tooltip'>" +
+                                     "<div class='arrow'></div>" +
+                                     "<h3 class='popover-header'></h3>" +
+                                     "<div class='popover-body'></div>" +
+                                     "<div class='popover-footer'>" +
+                                     "<button class='op-clear-history-btn btn btn-sm btn-secondary'" +
+                                     "type='button' title='Clear all history' disabled>Clear all</button></div>" +
```

Comment:
In other dialogs (like Select Metadata), every word in the button is capitalized, so to be consistent this should be "Clear All"